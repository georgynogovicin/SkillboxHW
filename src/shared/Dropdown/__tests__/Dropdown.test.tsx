import Enzyme, {shallow} from "enzyme";
import React from "react"
import {Dropdown} from ".."
import Adapter from "enzyme-adapter-react-16"

Enzyme.configure({adapter: new Adapter()})

describe('Dropdown', () => {
  test('should render', () => {
    const wrapper = shallow(<Dropdown children={<div></div>} button={<button>123</button>}/>)

    expect(wrapper).toBeDefined();
    expect(wrapper.find('div.container').isEmptyRender()).toBeFalsy();
  });

  test('should render (snapshot)', () => {
    const wrapper = shallow(<Dropdown children={<div></div>} button={<button>123</button>}/>)

    expect(wrapper).toMatchSnapshot()
  })
})