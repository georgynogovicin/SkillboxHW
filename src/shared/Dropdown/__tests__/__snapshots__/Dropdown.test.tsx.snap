// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dropdown should render (snapshot) 1`] = `
<div
  className="container"
>
  <div
    onClick={[Function]}
  >
    <button>
      123
    </button>
  </div>
</div>
`;
